src/PagesBody/ParticipantDataPage/ParticipantDataPage.js

```diff
@@ -0,0 +1,8 @@
+import ParticipantData from '../../Organisms/ParticipantData/ParticipantData';
+const randomParticipantID = 'ec7baebd-97c7-4ea3-8047-1305dc4bb243';
+
+const ParticipantDataPage = () => {
+  return <ParticipantData id={randomParticipantID} />;
```

Comment:
Use data passed in props or navigation or context, then fetch from dummy data by id